### PR TITLE
fix: improve rendering of private instructions

### PR DESCRIPTION
**Pull request by snitin315**
Resolve https://github.com/opencollective/opencollective/issues/5468
Frontend PR - https://github.com/opencollective/opencollective-frontend/pull/8000

Screenshot 2022-07-07 at 7 45 57 PM

### CODE REVIEW

File: templates/emails/ticket.confirmed.hbs [hunk 1]
```diff
@@ -26,9 +26,7 @@ Subject: {{ order.quantity }} {{pluralize "ticket" n=order.quantity}} confirmed
 
 {{#if event.data.privateInstructions}}
 <h2>Private Instructions</h2>
-<p>
-  {{event.data.privateInstructions}}
-</p>
+<pre>{{event.data.privateInstructions}}</pre>
```
Comment (Betree) [inline templates/emails/ticket.confirmed.hbs:29]: The styling is a bit rough. Can we try re-using something from another email? For example `templates/emails/collective.comment.created.hbs`:

```suggestion
<table style="width: 100%; margin-bottom: 30px;">
  <tbody>
    <tr>
      <td style="border: 1px solid #e8edee; border-radius: 6px; padding: 2em 1em;">
        <blockquote style="color: #798088;font-size: 13px;text-align: left;padding: 0px 1em;margin: 0em 0;white-space: pre-line;">{{event.data.privateInstructions}}</blockquote>
      </td>
    </tr>
  </tbody>
</table>
```

Could you try something along those lines, eventually play with the styles if there are issues, and leave a screenshot? Thanks!

Comment (snitin315) [reply templates/emails/ticket.confirmed.hbs:29]: @Betree This is how it looks now, and I feel it's looking good now.
Screenshot 2022-07-09 at 3 54 00 PM

**Review by Betree — APPROVED** [review]
Perfect!

**Comment by snitin315** [comment]
@RichardLitt yes, we should track it in a separate issue.

**Pull request merged**